docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+<!--- A brief description of this PR and the problem is trying to solve -->
+
+## Linked issue
+
+<!--- Put here the relevant issue from YouTrack -->
+
+## Type of change
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
+- [ ] 🛠 New feature (non-breaking change which adds functionality)
+- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] ⛑ git-flow chore (backport, hotfix, etc)
+
+## Developer checklist
+<!--- A mental checklist for the developer submitting the PR --->
+- [ ] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
+- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
+- [ ] I have updated any documentation accordingly, if needed.
```

Comment:
It might be helpful to give a list of the documentation sources that might need updating -- READMEs, cardanodocs.com, haddocks, etc.